src/components/tooltip/tooltip.spec.js

```diff
@@ -84,6 +84,31 @@ describe('MdTooltip Component', function() {
     expect(element.attr('aria-label')).toEqual('Tooltip');
   });
 
+  it('should not label the parent if it already has a label', function() {
+    buildTooltip(
+      '<md-button aria-label="Button Label">' +
+        '<md-tooltip md-visible="testModel.isVisible">' +
+          'Tooltip' +
+        '</md-tooltip>' +
+      '</md-button>'
+    );
+
+    expect(element.attr('aria-label')).toEqual('Button Label');
+  });
+
+  it('should not label the parent if it has already been labelled by ' +
```

Comment:
@clshortfuse Test for the presence of the `aria-labelledby` attribute on the parent element.